packages/camera/camera/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Fixes bug where calling a method after the camera was closed resulted in a Java `IllegalStateException` exception.
 * Fixes integration tests.
+* Fixes crash when streaming in iOS.
```

Comment:
Changes cannot be added to versions that have already been published.

Reply:
Fixed it.